srcalc/src/main/java/gov/va/med/srcalc/domain/model/Variable.java

```diff
@@ -11,6 +11,20 @@
     public static final int DISPLAY_NAME_MAX = 80;
     
     /**
+     * English description of the valid display name characters for readable
+     * error messages.
+     * @see #VALID_DISPLAY_NAME_REGEX
+     */
+    public static final String VALID_DISPLAY_NAME_CHARACTERS =
+            "letters, digits, spaces, and ~`!@#$%^&*()-_+=|\\.,<>/?'\":;";
+    
+    /**
+     * A regular expression that defines a valid variable key: {@value}
+     * @see #getDisplayName()
+     */
+    public static final String VALID_DISPLAY_NAME_REGEX = "[\\w ~`!@#$%^&*()-_+=|\\.,<>/?'\":;]+";
```

Comment:
Will we have to take any extra measures to escape any of these when displaying them?